### ISSUE TITLE
SmallPacket0x17 Error (New Build)

### ISSUE THREAD

**Issue by Wiggo32** [DarkstarProject/darkstar#4282]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 30171103_1

**_Server Version_** (type `!revision` in game) **:** 2fe3c089

**_Source Branch_** (master/stable) **:** master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Brand new build, brand new database, updated client and am seeing these errors pop up in game server window:
![smallpacket error](https://user-images.githubusercontent.com/30469395/33234298-dd53cea6-d1e9-11e7-8b88-a23a354d21c2.jpg)
Happens when zoning into another area sometimes. This came from zoning into Davoi.

(1024 is also the item ID for the Ghelsba Chest Key- I dont know if it means anything or not but there doesn't seem to be an explanation for https://github.com/DarkstarProject/darkstar/issues/4023)

**Comment by TeoTwawki** [comment]
This has been around for years
https://forums.dspt.info/viewtopic.php?f=15&t=21147&p=35332&hilit=SmallPacket0x17

**Comment by TeoTwawki** [comment]
1024 and 1025 were entity ids - appears that basically its checking NPC id and getting the player by mistake

**Comment by teschnei** [comment]
No, it's asking for the player (21828 and 21829) -  you see this a lot with
the BCNM cutscenes

**Comment by TeoTwawki** [comment]
>No, it's
You were too fast I edited

**Comment by TeoTwawki** [comment]
also I'm not entirely wrong: players are entities too.

**Comment by TeoTwawki** [comment]
I've changed it from warning to error for now since there are no known serious repercussions for that triggering.

**Comment by Wiggo32** [comment]
Here's some repro steps to get this issue to occur, still havent pinpointed why yet though. Maybe this will help trigger an idea:
While messing with Tebhi in Castle Oztroja (associated with BST AF2) I made him disappear like he's supposed to using npc:hideNPC() which worked fine with no warnings/errors.
THEN, I made him follow a path like he's supposed to using pathfind which works fine until you trade the Beast Collar. Once he disappears and is still following the path the Warning pops up:
![small packet](https://user-images.githubusercontent.com/30469395/35170015-4ee45c06-fd24-11e7-91f6-7c19a3d8d647.png)
